compiler/circle-opselector/CMakeLists.txt

```
@@ -0,0 +1,18 @@
+set(DRIVER "driver/Driver.cpp")
+
+file(GLOB_RECURSE SOURCES "pass/*.cpp")
+
+add_executable(opselector ${DRIVER} ${SOURCES})
```

executable name follows the module name.
```suggestion
add_executable(circle_opselector ${DRIVER} ${SOURCES})
```

---

Please use `circle-opselector` like `circle-tensordump`, `circle-verify`, The modules under `compiler` seem to use `-` for word separator.

---

OK :) Then we will change excutable name from 'opselector' to 'circle-opselector'.